nginx.conf

```diff
@@ -27,6 +27,8 @@ http {
   server {
     listen <%= ENV["PORT"] %>;
     server_name localhost;
+	
+	client_max_body_size 40M; #increase upload size limit
```

Comment:
The indentation threw me off a bit (this property can be lined up with the others above it), but should be good!